back-end/routes/pendingTransactions.js

```diff
@@ -0,0 +1,29 @@
+const express = require("express");
+const router = express.Router();
+const { PrismaClient } = require("@prisma/client");
+const prisma = new PrismaClient();
+
+router.get("/user", async (req, res) => {
```

Comment:
Rename route to getUserPendingTransactions.

Reply:
Let's change the route to /pendingTransactions/user?userId=<userId>